Update BAPI Document (#107)
SAP/cloud-sdk: docs/java/features/bapi-and-rfc/overview.mdx

</Tabs>

#### Set Destination as Environment Variable

> In order to your destination to be found in the local environment you need to supply destinations as an environment variable in your local machine:

Reviewer: ```suggestion
To let your destination be found in the local environment you need to supply an environment variable `destinations` on your local machine:
```

<Tabs groupId="os" defaultValue="windows" values={[ { label: 'Windows', value: 'windows', }, { label: 'Mac', value: 'mac', }]}>

<TabItem value="windows">

> You can set the destinations environment variable by the following command:

Reviewer: ```suggestion
You can set the `destinations` environment variable by the following command:
```
